.github/workflows/build-and-test.yaml

```diff
@@ -27,6 +27,12 @@ jobs:
         with:
           path: ./dist/*.tar.gz
 
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - run: |
+          pip install ./dist/curl_cffi-0.6.0b9.tar.gz
```

Comment:
how make the full version available in ci env? without overhead and extra scripts.

Reply:
You mean python versions?

Reply:
no, I mean `curl_cffi-0.6.0b9`.
hopefully pip supports glob path. resolved in https://github.com/yifeikong/curl_cffi/pull/235/commits/58339690d98cfffc9107096dd97b32367ef26ef6